Latex/inhalt/Abkürzungen.tex

```diff
@@ -1,7 +1,9 @@
-%Abkürzungsverzeichnis, Abkürzungen hier entsprechend des Beispiels eintragen und dann im Fließtext mit
-% \ac{Kürzel} nutzen
+%! Abkürzungsverzeichnis, Abkürzungen hier entsprechend des Beispiels eintragen
+%! und dann im Fließtext mit \ac{Kürzel} nutzen.
```

Comment:
Evtl. hier noch eine art hinweis?
```suggestion
%! Abkürzungsverzeichnis, Abkürzungen hier entsprechend des Beispiels eintragen
%! und dann im Fließtext mit \ac{Kürzel} nutzen.
%! Die Abkürzungen sollten lauf HAWA in alphabetischer Reihenfolge aufgelistet sein.
```
falls wir die suggestion übernehmen sollten, gucken dass dadurch der kommentar drüber nicht geändert wird ^^

Reply:
uff, das erklärt eigentlich dann das beleg dokument, ich frag mich überhaupt, warum das hier drin ist.

Reply:
noch is das aber nicht erlärt @DSczyrba  , das kannste danach entfernen, wenn nötig